styles/youtube/catppuccin.user.css

```diff
@@ -554,6 +554,10 @@
 
     /* Buttons */
 
+    .ytVideoMetadataCarouselViewModelHost {
```

Comment:
Do you mind moving this selector to line 1178, and adding it to the list of selectors already there for theming this from https://github.com/catppuccin/userstyles/pull/1483/files#diff-c08ba2f30a7396e0d274fcbc429a1616b4dd4373f815d47f2b082ef52dbf4928R1131? (Youtube is doing this werid thing where half of their classes start with `.yT` and the other half with `.Yt` now lol.)

https://github.com/catppuccin/userstyles/blob/615462f09e60c291c7aff5244b1f518bfa5d98a4/styles/youtube/catppuccin.user.css#L1178

Reply:
Sure :3